test/helpers/bibucketNock.ts

```diff
@@ -0,0 +1,732 @@
+    {
+      user: {
+        id: '{9d65d517-4898-47ac-9d2f-fd902d25d9f6}',
+        login: 'landtuna',
+        url: 'https://bitbucket.org/%7B9d65d517-4898-47ac-9d2f-fd902d25d9f6%7D/',
+      },
+      url: 'https://bitbucket.org/pypy/pypy/pull-requests/1',
+      body: 'Added a floor() ufunc to micronumpy',
+      createdAt: '2011-06-22T19:44:39.555192+00:00',
+      updatedAt: '2011-06-23T13:52:30.230741+00:00',
+      closedAt: null,
+      mergedAt: null,
+      state: 'DECLINED',
+      id: 1,
+      base: {
+        repo: {
+          url: 'https://bitbucket.org/pypy/pypy',
+          name: 'pypy',
+          id: '{54220cd1-b139-4188-9455-1e13e663f1ac}',
+          owner: { id: undefined, login: 'pypy', url: 'www.bitbucket.org/pypy' },
+        },
+      },
+    },
+  ],
+  totalCount: 1,
+  hasNextPage: true,
+  hasPreviousPage: false,
+  page: 1,
+  perPage: 1,
+};
+
+export const getPullRequestResponse = {
+  user: { id: '{9d65d517-4898-47ac-9d2f-fd902d25d9f6}', login: 'landtuna', url: undefined },
+  url: 'https://bitbucket.org/pypy/pypy/pull-requests/1',
+  body: 'Added a floor() ufunc to micronumpy',
+  createdAt: '2011-06-22T19:44:39.555192+00:00',
+  updatedAt: '2011-06-23T13:52:30.230741+00:00',
+  closedAt: undefined,
+  mergedAt: null,
+  state: 'DECLINED',
+  id: 1,
+  base: {
+    repo: {
+      url: 'https://bitbucket.org/pypy/pypy',
+      name: 'pypy',
+      id: '{54220cd1-b139-4188-9455-1e13e663f1ac}',
+      owner: {
+        login: 'landtuna',
+        id: '{9d65d517-4898-47ac-9d2f-fd902d25d9f6}',
+        url: 'https://bitbucket.org/%7B9d65d517-4898-47ac-9d2f-fd902d25d9f6%7D/',
+      },
+    },
+  },
+};
+
+export const getPullCommits = {
+  items: [
+    {
+      sha: undefined,
+      commit: {
+        url: 'https://bitbucket.org/ashwinahuja/pypy/commits/f799951483319e5397d41019de50f8e07b01b04f',
+        message: 'Remove the duplicated (unnecessary items)',
+        author: { name: 'ashwinahuja', email: 'undefined', date: '2018-09-13T16:19:32+00:00' },
+        tree: {
+          sha: 'f799951483319e5397d41019de50f8e07b01b04f',
+          url: 'https://bitbucket.org/ashwinahuja/pypy/commits/f799951483319e5397d41019de50f8e07b01b04f',
+        },
+        verified: false,
+      },
+    },
+    {
+      sha: undefined,
+      commit: {
+        url: 'https://bitbucket.org/ashwinahuja/pypy/commits/0e3d572c47c60df4760e541da6a05e5e305d6175',
+        message: 'Making datetime objects more compatible with old C extensions written for CPython',
+        author: { name: 'ashwinahuja', email: 'undefined', date: '2018-09-13T16:14:59+00:00' },
+        tree: {
+          sha: '0e3d572c47c60df4760e541da6a05e5e305d6175',
+          url: 'https://bitbucket.org/ashwinahuja/pypy/commits/0e3d572c47c60df4760e541da6a05e5e305d6175',
+        },
+        verified: false,
+      },
+    },
+  ],
+  totalCount: 2,
+  hasNextPage: false,
+  hasPreviousPage: false,
+  page: 1,
+  perPage: 2,
+};
+
+export const getIssuesResponse = {
```

Comment:
Is this const necessary sice we have these clases above?

Reply:
It is a real service response? It should be moved to `__MOCKS__` folder as the Github responses are.